docs/program-setup/serial/serial_communication.md

```diff
@@ -20,7 +20,29 @@ This program prints a "Hello World!" message that you can view on a serial termi
 
 Serial terminals run on your host PC. They provide an interface where your Mbed board can print and where you can type characters back to your board.
 
-Mbed CLI provides a serial terminal that is configured with a default baud rate of `9600`. When a single board is connected, run:
+Mbed CLI provides a serial terminal that is configured with a default baud rate of `9600`. You can either set it to [open after compilation](#opening-a-serial-terminal-after-compilation), or open it [manually](#manually-opening-a-serial-terminal).
+
+### Opening a serial terminal after compilation
+
+To compile an application, flash it onto a board and open a serial terminal:
+
+```
+mbed compile -t <TOOLCHAIN> -m <TARGET> --flash --sterm
+```
+
+This starts message printing (`Hello World!` in the example) once the application is flashed onto the board.
+
+To close the serial terminal, enter <kbd>Ctrl + C</kbd>.
+
+<span class="notes">**Notes:**
+- If your application uses a baud rate other than 9600, specify it with `-b <BAUDRATE>` in the command above.
+- This method only works if _one_ board of the TARGET you specify is connected. To work with multiple boards, open a serial terminal manually as described below.</span>
```

Comment:
```suggestion
<br/>- This method only works for a single board. To work with multiple boards, open a serial terminal manually as described below.</div>
```

Reply:
I removed the bit about the target because it implied you could make this work for multiple boards of different targets, and we don't explain that.

Reply:
As a user this will annoy me. You send me to "Manually opening a serial terminal" which then sends me to "Additional options" which explains how to connect multiple boards. Is there a way we can make this smoother? Since the two things explained in "Additional options" are how to use multiple boards and how to specify a baud rate (which you explained above), I think we can maybe we can do away with that bit. The `--help` bit can then have its own title or just be mentioned as a tip (we don't have to show output for `--help`).

Reply:
I agree we drop the "Additional options" section and replace it with a tip which is more compact.